doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -60,27 +64,33 @@ To enable cipher modes, set one or more of the following Kconfig options:
-+=======================+=====================================================================+======================================================================+
-| ECB no padding        | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_ECB_NO_PADDING_CC3XX` | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_ECB_NO_PADDING_OBERON` |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| CBC no padding        | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_CBC_NO_PADDING_CC3XX` | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_CBC_NO_PADDING_OBERON` |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| CBC PKCS#7 padding    | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_CBC_PKCS7_CC3XX`      | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_CBC_PKCS7_OBERON`      |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| CFB                   | Not supported                                                       | Not supported                                                        |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| CTR                   | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_CTR_CC3XX`            | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_CTR_OBERON`            |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| OFB                   | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_OFB_CC3XX`            | Not supported                                                        |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| XTS                   | Not supported                                                       | Not supported                                                        |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| Stream cipher         | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_STREAM_CIPHER_CC3XX`  | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_STREAM_CIPHER_OBERON`  |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
+You can use the following table to check the cipher algorithm support of each driver:
```

Comment:
```suggestion
Cipher algorithm support for each driver:
```

It is obvious to the reader what the table can be used for I believe.

Reply:
I followed the previous phrasing here but I can just update this one, the tech writer will decide this in the end anyway.